tests/test_madlib.py

```diff
@@ -1,14 +1,13 @@
 import pytest
-from madlib_cli.madlib import read_template, parse_template, merge
+from madlib import read_template, parse_template, merge
 
 
 def test_read_template_returns_stripped_string():
-    actual = read_template("assets/dark_and_stormy_night_template.txt")
+    actual = read_template("./assets/dark_and_stormy_night_template.txt")
```

Comment:
no `./` necessary. On execution, pytest and Python will search from the project root, so `"assets/dark_and_stormy_night_template.txt"` is the correct path for this file